launch_snakemake.py

```diff
@@ -270,11 +280,11 @@ def analysis_main(args, output, runnormal=False, normalname=False, normalfastqs=
             analysisdict["reference"] = "hg38"
             if tumorname:
                 if normalname:
-                    analysisdict["resultdir"] = f'{config["resultdir_hg38"]}/{basename_output}' #Use f'{config["testresultdir"]}/{tumorname}'for testing
+                    analysisdict["resultdir"] =  f'{config["resultdir"]}/{tumorname}' #f'{config["resultdir_hg38"]}/{basename_output}' #Use f'{config["testresultdir"]}/{tumorname}'for testing
```

Comment:
did we remove everything to do with hg19?

Reply:
The naming here should be f'{config["resultdir_hg38"]} so I recorrected the launch_snakemake script . Did that answer your question?